recipes/fastmlst/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=/tmp/record.txt
```

Comment:
Why is this needed? I see you tried to embed it in meta.yaml earler, but I don't see why that did not work. I'm alos not sure why --record and --single-version-externally-managed are necessary

Reply:
Hello! Thank you for your comment @bwlang 

I added `--single-version-externally-managed --record=/tmp/record.txt ` because without that line the local test generated an error `RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/biopython/`

I realized that by removing the build.sh file and adding `as` to the meta.yaml the local test works.

Do you think that is better to add all the installation logic to the meta.yaml?
If you think so, I could add a new update to the recipe with that change

Reply:
Maybe it's best to list the dependencies in the meta.yaml and avoid the need for this build.sh script?
looks like it 's just 'tqdm', 'pandas', 'biopython' right?

Reply:
I listed the dependencies ´tqdm, pandas, biopython´ in requirements/run in the meta.yaml. That looks fine?

I'm going to update the recipe without build.sh

Reply:
sounds like it should work nicely - i'll look for the update, (unless someone else gets to it first ;)